i18n-msg.html

```diff
@@ -64,166 +65,202 @@
+                 *
+                 * @property language
+                 * @type string
+                 * @default null
+                 * @readonly
+                 */
+                language: {
+                    type: String,
+                    value: null,
+                    observer: 'languageChanged'
+                }
+            },
+
+            /**
+             * The folder name where the localized `<lang>.json` files are located.
+             *
+             * @attribute messagesUrl
+             * @type string
+             * @default 'locales'
+             */
+            messagesUrl: 'locales',
+
+            /**
+             * An object containing the set of known language locales that have been loaded.
+             *
+             * @property locales
+             * @type object
+             * @default {}
+             */
+            locales: I18nMsg.locales, // static
+
+            ready: function () {
+                this.language = I18nMsg.lang;
+                this.messagesUrl = I18nMsg.url;
+
+                // Have instances observe window.I18nMsg.lang changes
+                // and go fetch the localized messages.json file.
+                var observerLang = new PathObserver(window.I18nMsg, 'lang');
+                observerLang.open(function (newValue, oldValue) {
+                    numInstancesUpdated_ = 0;
+                    this.language = newValue;
+                }.bind(this));
+
+                // Have instances observe window.I18nMsg.lang changes
+                // and go fetch the localized messages.json file.
+                var observerUrl = new PathObserver(window.I18nMsg, 'url');
+                observerUrl.open(function (newValue, oldValue) {
+                    numInstancesUpdated_ = 0;
+                    this.messagesUrl = newValue;
+
+                    if (!this.language || fetching_) {
+                        return;
+                    }
+
+                    this.fetchLanguage();
+                }.bind(this));
+
+                instances.push(this);
+            },
+
+            attached: function () {
+                if (!this.msgid) {
+                    this.msgid = this.innerHTML;
+                }
+
+                var msg = this.locales[this.language] && this.locales[this.language][this.msgid];
+                if (msg && msg.message) {
+                    this.innerHTML = msg.message;
+                }
+            },
+
+            languageChanged: function () {
+                // Don't fetch .json file if it has already been
+                // fetched or another instance is already trying to.
+                if (this.language && this.locales[this.language] && !fetching_) {
+                    // Send one signal that language changed to outside.
+                    if (numInstancesUpdated_ == instances.length - 1) {
+                        this.fire('i18n-language-ready', {language: this.language});
```

Comment:
Can you document this event. SEe https://github.com/PolymerElements/seed-element/blob/master/seed-element.html#L122-L127